java/test/jmri/jmrix/loconet/LnOpsModeProgrammerTest.java

```diff
@@ -22,6 +21,7 @@ public void testGetCanWriteAddress() {
         Assert.assertFalse("can write address", programmer.getCanWrite("1234"));
     }
 
+    /*
     @Test
     public void testSetMode() {
         try {
```

Comment:
Is these tests intended to be removed or just removed because the tests currently fails?
If the latter, then I suggest you remove the comment and instead add `@Ignore` to the tests.

```
    @Ignore   // Test currently fail due to changes in PR #13254. Needs to be fixed.
    @Test
    public void testSetMode() {
```